src/server/scripts/Northrend/Naxxramas/boss_gothik.cpp

```diff
@@ -374,13 +374,16 @@ class boss_gothik : public CreatureScript
             switch (spellInfo->Id)
             {
                 case SPELL_INFORM_LIVING_TRAINEE:
-                    me->SummonCreature(NPC_DEAD_TRAINEE, PosSummonDead[pos].GetPositionX(), PosSummonDead[pos].GetPositionY(), PosSummonDead[pos].GetPositionZ(), PosSummonDead[pos].GetOrientation());
+                    if(Creature* target = me->SummonCreature(NPC_DEAD_TRAINEE, PosSummonDead[pos].GetPositionX(), PosSummonDead[pos].GetPositionY(), PosSummonDead[pos].GetPositionZ(), PosSummonDead[pos].GetOrientation()))
+                        me->CastSpell(target, SPELL_INFORM_DEAD_TRAINEE, true);
```

Comment:
```suggestion
                    if(Creature* target = me->SummonCreature(NPC_DEAD_TRAINEE, PosSummonDead[pos].GetPositionX(), PosSummonDead[pos].GetPositionY(), PosSummonDead[pos].GetPositionZ(), PosSummonDead[pos].GetOrientation()))
                    {
                        me->CastSpell(target, SPELL_INFORM_DEAD_TRAINEE, true);
                    }
```

add the brackets even for the other code https://www.azerothcore.org/wiki/cpp-code-standards#brackets